test/test.kitchen.ts

```diff
@@ -54,7 +56,7 @@ it('should be able to use the d.ts', async () => {
   console.log(`${__filename} staging area: ${stagingPath}`);
   await spawnp('npm', ['pack']);
   const tarball = `${pkg.name}-${pkg.version}.tgz`;
-  await rename(tarball, `${stagingPath}/google-auth-library.tgz`);
+  await mvp(tarball, `${stagingPath}/google-auth-library.tgz`);
```

Comment:
Please add a brief comment explaining why we brought a module in to do this when fs.rename is right there :)